src/View/Helper/BreadcrumbsHelper.php

```diff
@@ -235,10 +235,14 @@ public function getCrumbs()
      * - *innerAttrs* To provide attributes in case your separator is divided in two elements.
      * All other properties will be converted as HTML attributes and will replace the *attrs* key in the template.
      * If you use the default for this option (empty), it will not render a separator.
-     * @return string The breadcrumbs trail
+     * @return string|null The breadcrumbs trail
      */
     public function render(array $attributes = [], array $separator = [])
     {
+        if (empty($this->crumbs)) {
```

Comment:
`empty()` eats a bunch of values where as `!== null` or `count($this->crumbs) === 0` are more specific.